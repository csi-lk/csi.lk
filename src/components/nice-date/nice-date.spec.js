import React from 'react'

import NiceDateComponent from './nice-date'

describe('Component: Nice Date', () => {
  let props
  beforeEach(() => {
    props = {
      date: '2012-10-01T00:00:00.000Z',
    }
  })
  it('renders', () => {
    const wrapper = shallow(<NiceDateComponent {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
