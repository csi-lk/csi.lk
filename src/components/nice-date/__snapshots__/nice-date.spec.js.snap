// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Nice Date renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NiceDateComponent
    date="2012-10-01T00:00:00.000Z"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "October 2012",
    },
    "ref": null,
    "rendered": "October 2012",
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "October 2012",
      },
      "ref": null,
      "rendered": "October 2012",
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
